Lesson1/src/HomeWorkApp.java

```diff
@@ -0,0 +1,41 @@
+public class HomeWorkApp {
+    public static void main(String[] args) {
+        printThreeWords();
+        checkSumSign();
+        printColor();
+        compareNumbers();
+    }
+    public static void printThreeWords(){
+        System.out.println("Orange\nBanana\nApple");
+    }
+    public static void checkSumSign(){
+        int a = 5;
+        int b = -30;
+
+        if(a+b < 0) {
+            System.out.println("Сумма отрицательная");
+        } else {
+            System.out.println("Сумма положительная");
+        }
+    }
+    public static void printColor(){
+        int value = 1000;
+
+        if (value <= 0) {
+            System.out.println("Красный");
+        } else if (value > 0 && value <= 100) {
```

Comment:
```suggestion
        else if (value <= 100)
```
поскольку данное условие работает само по себе из-за неактульности первого `value<= 0 -> false <=> value > 0`